src/annotator/sidebar.js

```diff
@@ -89,11 +89,9 @@ export default class Sidebar extends Delegator {
       if (config.theme === 'clean') {
         this.iframeContainer.classList.add('annotator-frame--theme-clean');
       } else {
-        const bucketBar = new BucketBar(
-          this.iframeContainer,
-          guest,
-          config.BucketBar
-        );
+        const bucketBar = new BucketBar(this.iframeContainer, guest, {
+          contentContainer: config.contentContainer || document.body,
+        });
```

Comment:
I would rather not overwrite the default value in the bucket bar constructor.
What about?
```suggestion
        const bucketBar = new BucketBar(this.iframeContainer, guest, {
          contentContainer: config.contentContainer
          });
```
Or even:
```suggestion
        const bucketBar = new BucketBar(this.iframeContainer, guest, config);
```

Reply:
I see a problem with passing `config` here because it ends up passing all of the sidebar's configuration options to the bucket bar. That could potentially have unintended consequences.

Reply:
I do agree about not overriding `BucketBar`'s defaults when no `contentContainer` is specified. That leaves the responsibility of how to observe "the whole (HTML) document scrolling" up to `BucketBar`, which makes more sense.